pkg/api/helpers.go

```diff
@@ -62,7 +62,13 @@ var Semantic = conversion.EqualitiesOrDie(
 	},
 )
 
-var standardResources = util.NewStringSet(string(ResourceMemory), string(ResourceCPU))
+var standardResources = util.NewStringSet(
```

Comment:
@vishh These new resources are going to be official kubernetes resources, and are valid in a ResourceList in the context of a ResourceQuota, but they aren't valid in the context of a Pod's or Container's limit.  Does that matter?  I can't tell your intent because ValidateResourceName isn't used yet AFAICS.